enterprise/backend/test/metabase_enterprise/serialization/v2/extract_test.clj

```diff
@@ -38,28 +39,33 @@
 
       (testing "a top-level collection is extracted correctly"
         (let [ser (serdes.base/extract-one "Collection" {} (select-one "Collection" [:= :id coll-id]))]
-          (is (= [{:model "Collection" :id coll-eid :label coll-slug}] (:serdes/meta ser)))
+          (is (schema= {:serdes/meta       (s/eq [{:model "Collection" :id coll-eid :label coll-slug}])
+                        :personal_owner_id (s/eq nil)
+                        :parent_id         (s/eq nil)
+                        s/Keyword          s/Any}
+                       ser))
           (is (not (contains? ser :location)))
-          (is (not (contains? ser :id)))
-          (is (nil? (:personal_owner_id ser)))
-          (is (contains? ser :parent_id))
-          (is (nil? (:parent_id ser)))))
+          (is (not (contains? ser :id)))))
```

Comment:
I couldn't find a way to enforce this negative check with Schema - is there a way?

Reply:
I don't think there's a "disallowed key" schema, at least not if you open up the map with `s/Keyword s/Any` or something like that. But you could do something like 

```clj
{:id (s/eq ::disallowed "this key should never be present")}
```

which would be obvious enough and do the trick I think.